docs/data_sources/epacems.rst

```diff
@@ -78,10 +92,18 @@ on GitHub for pointers on how to access this big dataset efficiently using :mod:
 PUDL Data Tables
 ^^^^^^^^^^^^^^^^
 
-Clicking on the links will show you the names and descriptions of the fields available
-in the CEMS table.
+Clicking on the links will show you a description of the table as well as the names and
+descriptions of each of its fields.
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Data Dictionary
+     - Browse Online
+   * - :ref:`hourly_emissions_epacems`
+     - https://data.catalyst.coop/pudl/hourly_emissions_epacems
```

Comment:
The EPA CEMS isn't available via the Datasette instance and I don't think we intend for it to be, so this link is dead.